services/blockchain-app-registry/shared/utils/downloadRepository.js

```diff
@@ -112,10 +114,12 @@ const getFileDownloadURL = async (file) => {
 	}
 };
 
+// TODO: Update implementation to handle undefined lastSyncedCommitHash
```

Comment:
Please create an issue (bug) for this.

Reply:
https://github.com/LiskHQ/lisk-service/issues/1500